tests/api/entities/contributions.test.js

```diff
@@ -76,6 +77,24 @@ describe('entities:contributions', () => {
     getWorkId(patches2[0]._id).should.equal(workB._id)
     getWorkId(patches2[1]._id).should.equal(work._id)
   })
+
+  describe('filter', () => {
+    it('should filter by claim property', async () => {
+      const user = await getReservedUser()
+      const work = await createWork({ user })
+      const property = 'wdt:P921'
+      await updateClaim({ uri: work.uri, property, newValue: 'wd:Q1', user })
+      await updateClaim({ uri: work.uri, property: 'wdt:P136', newValue: 'wd:Q2', user })
+      await updateClaim({ uri: work.uri, property, newValue: 'wd:Q3', user })
+      await updateClaim({ uri: work.uri, property, oldValue: 'wd:Q1', user })
```

Comment:
another way to explicitly say "property could be labels" (cf last comment) could be to add a test also